subprojects/android-test/test-app-kaspresso/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
No need. We have it in the root `.gitgnore` config

Reply:
Didn't notice, removed the file